### DISCUSSION TITLE
Question 2.1.3 Multiplayer

### DISCUSSION THREAD

**Discussion by gamerabbit16** [MinicraftPlus/minicraft-plus-revived#652] [Q&A] [answered]
Do the multiplayer work again on the latest version (2.1.3)

**Comment by Makkkkus** [comment, marked as answer]
Sadly not. Multiplayer was removed in 2.0.7, but a slight outdated version is available here: https://www.github.com/ElVir-Software/minicraft-plus-online.

**Comment by ImRichard365** [comment]
reasons?

**Comment by BenCheung0422** [comment]
The multiplayer system is not complete and is buggy.
